lib/handler/message_reaction_add.go

```diff
@@ -0,0 +1,16 @@
+package handler
+
+import (
+	"github.com/tpc3/DuckPolice/lib/config"
+
+	"github.com/bwmarrin/discordgo"
+)
+
+func MessageReactionAdd(session *discordgo.Session, reaction *discordgo.MessageReactionAdd) {
+	msg, _ := session.ChannelMessage(reaction.ChannelID, reaction.MessageID)
+
+	if reaction.UserID != session.State.User.ID && reaction.Emoji.Name == config.CurrentConfig.Duplicate.Delete {
```

Comment:
According to discord document, `reaction.Emoji.Name` can be null.
https://discord.com/developers/docs/resources/emoji#emoji-object

Reply:
I don't know how to deal with this, so for now, I have made a code that returns return in case of null.

Reply:
You can check both of `id` and `name`.
So, `reaction.Emoji.ID == config.CurrentConfig.Duplicate.Delete || reaction.Emoji.Name == config.CurrentConfig.Duplicate.Delete` is better.

Reply:
Thanks